microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -218,14 +298,48 @@ webView->NavigateToString(htmlString);
 
 
 <!-- ====================================================================== -->
-## Virtual host name mapping
+## Loading local content by using virtual host name mapping
 
 Another way to load local content in a WebView2 control is to use virtual host name mapping.  This involves mapping a local domain name to a local folder, so that when the WebView2 control attempts to load a resource for that domain, it will load the content from the specified local folder location instead. The origin of the document will also be the virtual host name. 
 
 Due to a current limitation, media files that are accessed using a virtual host name can be slow to load.<!-- when limitation is fixed, remove sentence -->
 
 This approach lets you specify the cross-origin access, by using the `CoreWebView2HostResourceAccessKind` enum.
```

Comment:
```suggestion
This approach lets you specify the cross-origin access, by using the `CoreWebView2HostResourceAccessKind` enum.

Due to a current limitation, media files that are accessed using a virtual host name can be slow to load.<!-- when limitation is fixed, remove sentence -->
```
better flow by moving limitation to bottom.
@david-risney - is there a better place for this limitation down in an h6 section, or keep up here?